<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>599</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Wi Motor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#side_menu {
	background-color: rgb(30, 30, 30);
}

#wimotor {
	padding: 0 24px;
	color: rgb(255, 255, 255);
	font-size: 24px;
	font-weight: bold;
}

#side_menu QPushButton {
	padding: 8px 12px;
	font-size: 14px;
	text-align: left;
	color: rgb(255, 255, 255);
	border: none;
}

#side_menu QPushButton:checked {
	padding-left: 8px;
	background-color: rgb(73, 0, 131);
	border-left: 4px solid rgb(133, 0, 237);
}

#side_menu QPushButton:hover {
	background-color: rgb(88, 0, 169) !important;
}

#main_page {
	background-color: rgb(240, 240, 240);
}

QScrollArea {
	border: none;
}

#bon_page_label,
#mekanik_page_label {
	font-size: 24px;
	color: rgb(150, 150, 150);
}

#cari_barang_label {
	font-size: 14px;
	color: rgb(30, 30, 30);
}

#cari_barang_input {
	padding: 4px 8px;
	font-size: 14px;
	color: rgb(30, 30, 30);
}

#cari_barang_input:focus {
    border: 1px solid rgb(88, 0, 169);
}

#tambah {
	padding: 5px 0;
	font-size: 14px;
	color: rgb(255, 255, 255);
	background-color: rgb(88, 0, 169);
	border: 1px solid  rgb(88, 0, 169);
}

#tambah:hover {
	background-color: rgb(69, 0, 133);
}

#pilih {
	padding: 5px 0;
	font-size: 14px;
	color: rgb(88, 0, 169);
	background-color: transparent;
	border: 1px solid  rgb(88, 0, 169);
}

#pilih:hover {
	background-color: rgb(215, 215, 215);
}

#table_header {
	background-color: rgb(30, 30, 30);
}

#table_header QLabel {
	font-size: 14px;
	color: rgb(255, 255, 255);
}</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="side_menu" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>24</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>24</number>
       </property>
       <item>
        <widget class="QLabel" name="wimotor">
         <property name="text">
          <string>Wi Motor</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>36</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="menu_list" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="gudang">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Gudang</string>
            </property>
            <property name="icon">
             <iconset resource="assets/resources.qrc">
              <normaloff>:/Menu/Warehouse.png</normaloff>:/Menu/Warehouse.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bon">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Bon</string>
            </property>
            <property name="icon">
             <iconset resource="assets/resources.qrc">
              <normaloff>:/Menu/Recipe.png</normaloff>:/Menu/Recipe.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mekanik">
            <property name="text">
             <string>Mekanik</string>
            </property>
            <property name="icon">
             <iconset resource="assets/resources.qrc">
              <normaloff>:/Menu/Mechanic.png</normaloff>:/Menu/Mechanic.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="main_page">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="gudang_page">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>802</width>
             <height>599</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>32</number>
            </property>
            <property name="leftMargin">
             <number>24</number>
            </property>
            <property name="topMargin">
             <number>24</number>
            </property>
            <property name="rightMargin">
             <number>24</number>
            </property>
            <property name="bottomMargin">
             <number>24</number>
            </property>
            <item>
             <widget class="QWidget" name="gudang_header" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>16</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="cari_barang_label">
                 <property name="text">
                  <string>Cari barang</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="cari_barang_input">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="tambah">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Tambah</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pilih">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Seleksi</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="gudang_body" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="table_header" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,2,0,0">
                  <property name="spacing">
                   <number>32</number>
                  </property>
                  <property name="leftMargin">
                   <number>8</number>
                  </property>
                  <property name="topMargin">
                   <number>11</number>
                  </property>
                  <property name="rightMargin">
                   <number>8</number>
                  </property>
                  <property name="bottomMargin">
                   <number>11</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="no">
                    <property name="minimumSize">
                     <size>
                      <width>40</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>40</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>No</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="kode">
                    <property name="minimumSize">
                     <size>
                      <width>90</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>160</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Kode barang</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="nama">
                    <property name="minimumSize">
                     <size>
                      <width>250</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Nama barang</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="kuantiti">
                    <property name="minimumSize">
                     <size>
                      <width>70</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>70</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Kuantiti</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="kelola">
                    <property name="minimumSize">
                     <size>
                      <width>160</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>160</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Kelola</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="table_body">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="bon_page">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>16</number>
        </property>
        <property name="topMargin">
         <number>16</number>
        </property>
        <property name="rightMargin">
         <number>16</number>
        </property>
        <property name="bottomMargin">
         <number>16</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="bon_page_label">
          <property name="text">
           <string>Halaman belum tersedia 1</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mekanik_page">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>153</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="mekanik_page_label">
          <property name="text">
           <string>Halaman belum tersedia 2</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>153</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
